docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1501,7 +1501,7 @@ Search for an image on [Docker Hub](https://hub.docker.com).
 
 **Example request**:
 
-    GET /images/search?term=sshd HTTP/1.1
+    GET /v1.18/images/search?term=sshd HTTP/1.1
```

Comment:
We should add that to the release-checklist (there's a section in there that explains bumping the API version)